python/bin/p1_lband_extract

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+
+from fusion_engine_client.utils.trace import HighlightFormatter, BrokenPipeStreamHandler
+from fusion_engine_client.utils.time_range import TimeRange
+from fusion_engine_client.utils.log import locate_log, DEFAULT_LOG_BASE_DIR
+from fusion_engine_client.utils.argument_parser import ArgumentParser, ExtendedBooleanAction
+from fusion_engine_client.utils import trace as logging
+from fusion_engine_client.parsers import MixedLogReader
+from fusion_engine_client.messages import *
+import os
+import sys
+
+# Add the Python root directory (fusion-engine-client/python/) to the import search path to enable FusionEngine imports
+# if this application is being run directly out of the repository and is not installed as a pip package.
+root_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(0, root_dir)
+
+root_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(root_dir)
```

Comment:
Huh this is in p1_print, so I assume it was just copy/pasted here, but not sure why we have both prepend (insert) and append? Also we're doing the same root_dir statement twice. I think the append was just accidental leftovers after https://github.com/PointOneNav/fusion-engine-client/commit/57ddee92272bbae015203332c4ec32c662068748?